supabase/.env.exemple.redis

```
@@ -0,0 +1,17 @@
+STRIPE_WEBHOOK_SECRET=test
+STRIPE_SECRET_KEY=test
```

```suggestion
STRIPE_WEBHOOK_SECRET=whsec_sample
STRIPE_SECRET_KEY=sk_test_sample
```

to make it easier to know that webhook secret starts with `whsec_` and key starts with `sk_test_`

---

I do not understand this change. what `whsec` are you talking about?

---

I have not touched stripe, why would I change this?

---

It appears as a new change in the PR

The reason for using `whsec_` and `sk_test` is that is the way that these keys are provided for Strapi.

In this case, can you do
```suggestion
STRIPE_WEBHOOK_SECRET=test # whsec_....
STRIPE_SECRET_KEY=test # sk_test_...
```

This is to improve DX, the developers will know which key they need to choose even if they don't have that much expertise in Stripe